### DISCUSSION TITLE
2023-09-18-open

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#156] [General]
# 该以一种怎么样的心态来对待开源 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share最近好几位我喜欢的开发者被开源工作影响到了心情，加上国内的开源环境氛围不是很好，想碎碎念来聊一下，对于开发者和使用者，我们应该以一种怎么样的心态来对待开源，让自己可以持续的快乐写自己喜欢的代码。可能有些我理解偏差地方，很欢迎大伙指出。

https://tw93.fun/2023-09-18/open.html

**Comment by sangxiaoyu** [comment]
您好博主，我是公众号《桑小榆呀》号主。此篇文章我觉得很好让大多数人有了深刻的思考以及帮助。因此我想转载分享，希望能获得您的允许，谢谢！

**Reply by tw93** [reply]
标注来源即可

**Reply by sangxiaoyu** [reply]
好勒，谢谢

**Comment by Jie-Huangi** [comment]
博主大大，我fork了你的个人主页。真的是因为特别喜欢，不想错过这样的主题。刚开始copy过来的时候一股脑的把你的资料全删了。事后想想应该留下原作者的足迹。现在已经在readme里面添加参考链接。感谢博主的开源代码，十分感谢。

**Reply by tw93** [reply]
客气

**Comment by jiangyangcreate** [comment]
之前总想憋个大招，期许很高，昨天在线下听tw93老师的讲解，感觉忽然没有那么拧巴了。

”大家都轻松些，松弛些，写开源和用开源都是为了我们工程师那种独特的 Happy，不需要较真。“这句话真的很好。
